### PR TITLE
reference/cdc: add usage document for ticdc

### PR DESCRIPTION
**Pull request by amyangfei**
### What is changed, added or deleted? (Required)

Add document for TiCDC

### Which TiDB version(s) do your changes apply to? (Required)
master, v4.0.0-beta.2

- [x] master (the latest development version, including v4.0 changes for now)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @leoppro @TomShawn

File: reference/tools/ticdc/deploy.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+$ cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_2.log --status-addr=127.0.0.1:8302
+$ cdc server --pd=http://10.0.10.25:2379 --log-file=ticdc_3.log --status-addr=127.0.0.1:8303
+```
+
+## 第 2 步：创建同步任务
+
+假设需要将上游所有的库表（系统表除外）同步到下游的 MySQL，可以通过以下命令创建同步任务
+
+```
+$ cdc cli changefeed create --pd=http://10.0.10.25:2379 --start-ts=415238226621235200 --sink-uri="mysql://root:123456@127.0.0.1:3306/"
```
Comment (leoppro) [inline reference/tools/ticdc/deploy.md]: 建议添加 sink 格式的说明，参考https://www.cockroachlabs.com/docs/stable/create-changefeed.html#sink-uri

Comment (leoppro) [reply reference/tools/ticdc/deploy.md]: 是否应该添加 kafka 输出的格式说明以及约束说明？

Comment (amyangfei) [reply reference/tools/ticdc/deploy.md]: Good catch, added

File: reference/tools/ticdc/sink.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+```
+--sink-uri="kafka://127.0.0.1:9092/cdc-test?kafka-version=2.4.0&partition-num=6&max-message-bytes=67108864&replication-factor=1"
+```
+
+参数解析
+
+| 参数               | 解析                                                         |
+| ------------------ | ------------------------------------------------------------ |
+| 127.0.0.1          | 下游 kafka 对外提供服务的 IP                                 |
+| 9092               | 下游数据密码                                                 |
```
Comment (leoppro) [inline reference/tools/ticdc/sink.md]:
```suggestion
| 9092               | 下游 kafka 链接端口                                                 |
```

**Comment by leoppro** [comment]
LGTM

**Pull request merged**